R/step_geohash.R

```diff
@@ -0,0 +1,143 @@
+
```

Comment:
While annoying, I think we can recreate the `check_string()` and `check_type()` by fetching them from the namespace instead 

```r
check_string <- getFromNamespace("check_string", "recipes")
check_type <- getFromNamespace("check_type", "recipes")
```

Reply:
Oh. Great, this is why I added deps.